Examples/UIExplorer/js/LayoutAnimationExample.js

```diff
@@ -30,12 +30,22 @@ const {
   Text,
   View,
   TouchableOpacity,
+  Platform,
+  UIManager
 } = ReactNative;
 
 class AddRemoveExample extends React.Component {
-  state = {
-    views: [],
-  };
+  state: any;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      views: [],
+    };
+    if (Platform.OS === 'android') {
+      UIManager.setLayoutAnimationEnabledExperimental && UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
Probably better to move it up and not do in constructor since this is a global config anyways. Also the `Platform` check is unnecessary.